tests/Unit/Renderer/Twig/Extensions/AssetsTest.php

```diff
@@ -30,18 +33,33 @@ public function testGetAsset()
     {
         /** @var UrlGenerator|MockObject $urlGenerator */
         $urlGenerator = $this->createMock(UrlGenerator::class);
+        /** @var AssetsProvider|MockObject $assets */
```

Comment:
Eigentlich eher sowas
```suggestion
        /** @var AssetsProvider&MockObject $assets */
```